utils/githooks/pre-commit.d/20-isort.sh

```diff
@@ -1,17 +1,25 @@
-#!/bin/sh
+#!/bin/bash
 
 # Runs isort for the DAOS project.
 
 set -ue
 
-if ! command -v isort > /dev/null 2>&1
-then
+if ! command -v isort > /dev/null 2>&1; then
     echo "No isort checking, install isort command to improve pre-commit checks"
+    . /etc/os-release
+    if [ "$ID" = "fedora" ]; then
+        echo "dnf install python3-isort"
```

Comment:
Just FYI we should keep an eye on this. Installing `python3-isort` might not give the right isort version. But the version from requirements.txt might not be available either so the process is already not solid.